zhaquirks/thirdreality/plug.py

```diff
@@ -0,0 +1,53 @@
+"""Third Reality Plug devices."""
+
+from typing import Final
+
+from zigpy.quirks.v2 import ClusterType, add_to_registry_v2
+import zigpy.types as t
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+from zhaquirks import CustomCluster
+from zhaquirks.thirdreality import THIRD_REALITY
+
+
+class ThirdRealityPlugCluster(CustomCluster):
+    """The private cluster id of Third Reality's PLUG."""
+
+    cluster_id = 0xFF03
+
+    class AttributeDefs(BaseAttributeDefs):
+        """Attribute of private cluster id."""
+
+        reset_summation_delivered: Final = ZCLAttributeDef(
+            id=0x0000,
+            name="reset_summation_delivered",
+            type=t.uint8_t,
+            is_manufacturer_specific=True,
+        )
+
+
+(
+    add_to_registry_v2(THIRD_REALITY, "3RSP02028BZ")

```

Comment:
You can use `.also_applies_to(THIRD_REALITY, "3RSPE01044BZ")` to make this quirk apply for both plugs.
The second quirk below can be removed then.